x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/reducer.ts

```diff
@@ -45,7 +45,6 @@ export const initialPolicyDetailsState: () => Immutable<PolicyDetailsState> = ()
     total: 0,
     other: 0,
   },
-  license: undefined,
```

Comment:
Is this intentional change? Does it affect something?

Reply:
it's intentional, when initializing, we shouldn't attempt to clear out any previously loading license information.  I think this should be refactored so that we have license info elsewhere in state and not associated with a particular page/feature store.  However, I wanted to make the change small since this needs to be backported in 7.11.